src/Identity/UI/src/IdentityBuilderUIExtensions.cs

```diff
@@ -36,5 +63,96 @@ public static IdentityBuilder AddDefaultUI(this IdentityBuilder builder)
 
             return builder;
         }
+
+        private static Assembly GetApplicationAssembly(IdentityBuilder builder)
```

Comment:
cc @captainsafia @halter73 this is the other piece of code in the framework that does a bad thing 😄